src/flatc.cpp

```diff
@@ -221,6 +221,8 @@ const static FlatCOption options[] = {
     "Only generated one typescript file per .fbs file." },
   { "", "annotate", "SCHEMA",
     "Annotate the provided BINARY_FILE with the specified SCHEMA file." },
+  { "", "no-leak-private-annotation", "",
+    "Prevents multiple type of annotations within a Fbs SCHEMA file." },
```

Comment:
Please add a comment saying this is required for generating private types for Rust.